src/App.js

```diff
@@ -46,10 +46,14 @@ function App(props) {
             exact
             component={localStorage.getItem("token") ? CreateVideo : Signup}
           />
-          <Route exact path="/users/:username" component={UserPage} />
+          <PrivateRoute
+            path="/users/:username"
+            exact
+            component={localStorage.getItem("token") ? UserPage : Login}
+          />
```

Comment:
We actually don't want profiles to be private, we want them to be public so you can send people to your profile or have others visit naturally. We also probably don't want to do any token checking here.

Reply:
Done. I changed it back to a regular route. I'm trying to wrap my head around what you're saying because currently there is no way for a user to access anyone else's profile page and the "Profile" button is currently set solely for your page (hence why I figured it would be a private route that you can only access once you're logged in). I would imagine there would possibly be a future feature change where you could find other users through the "Browse" section along with videos.